### ISSUE TITLE
Allow multiple selection of fields

### ISSUE THREAD

**Issue by rerpha** [ess-dmsc/nexus-constructor#380]
Currently only one field can be selected. I have tried this and it seems like it's a bit more involved than just applying a `QAbstractItemView::SelectionMode` (for example `ExtendedSelection`, which is what we want)

**Comment by rerpha** [comment]
this is blocked by #359

**Comment by rerpha** [comment]
how important is this? is it likely that components will have lots and lots of fields? 

if not, i don't think this should be a priority.

**Comment by matthew-d-jones** [comment]
We will have a better idea whether this is important once we start using the application properly, which should be soon. Marking this low priority for now.

**Comment by kmurica** [comment]
@matthew-d-jones Any news on the importance of this issue? I am a bit unsure what the benefit is here with the functionality of selecting multiple fields simultaneously. Although, my experience with the nexus constructor so far is limited. :)

**Comment by matthew-d-jones** [comment]
Possibly useful when deleting multiple fields, but I'm not sure how often you'd want to do that. I think it best to close the ticket.
